From abad46e5a220895b8b55d4c8e975dae42a3a340b Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Thu, 2 Nov 2023 21:07:14 +0100
Subject: [PATCH] setup

---
 setup.cfg | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index b24bd12..12de125 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -54,16 +54,9 @@ packages =
 	epics
 	epics.autosave
 	epics.compat
-	epics.clibs
 	epics.devices
 	epics.qt
 	epics.wx
-	epics.clibs.darwin64
-	epics.clibs.linux64
-	epics.clibs.linux32
-	epics.clibs.linuxarm
-	epics.clibs.win64
-	epics.clibs.win32
 install_requires = 
 	setuptools
 	numpy
