From ddfe6f9067648e1018b0be538090f6b0df2ba2ce Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Wed, 12 Jun 2024 12:41:53 +0200
Subject: [PATCH] [PATCH] setup

---
 setup.cfg | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 923cf76..61131d6 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -54,17 +54,9 @@ packages =
 	epics
 	epics.autosave
 	epics.compat
-	epics.clibs
 	epics.devices
 	epics.qt
 	epics.wx
-	epics.clibs.darwin64
-	epics.clibs.darwinarm64
-	epics.clibs.linux64
-	epics.clibs.linux32
-	epics.clibs.linuxarm
-	epics.clibs.win64
-	epics.clibs.win32
 install_requires = 
 	setuptools
 	numpy
