From 6f826affe0dc5a8b0d9b7754bac705a4ea083fa2 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Thu, 2 Nov 2023 20:23:13 +0100
Subject: [PATCH] setup.patch

---
 setup.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 7938f86..019d722 100644
--- a/setup.py
+++ b/setup.py
@@ -59,9 +59,7 @@
                       'Programming Language :: Python',
                       'Topic :: Scientific/Engineering'],
       packages = ['epics','epics.wx','epics.devices', 'epics.compat',
-                  'epics.autosave', 'epics.clibs'],
-      package_data = {'epics.clibs': ['darwin64/*', 'linux64/*', 'linux32/*',
-                                      'linuxarm/*', 'win32/*', 'win64/*']},
+                  'epics.autosave'],
       install_requires = ['setuptools']
       )
 
